docs/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md

```diff
@@ -12,3 +12,7 @@ import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 In addition to Stock Kubernetes (which could be deployed on cloud or on-premise), Camunda only officially tests and supports the following platforms:
 
 <DocCardList items={useCurrentSidebarCategory().items}/>
+
+:::caution Web Modeler
+We do not guarantee that Web Modeler Beta works on these cloud platforms and currently offer no dedicated support for them. However, it is likely that Web Modeler Beta will just work.
```

Comment:
I understand what you're saying here, but "it is likely that Web Modeler Beta will just work" is a bit vague. I would likely switch the sentence around so it reads something like:
"While it is likely Web Modeler Beta will work on your cloud platform, we do not guarantee functionality and currently offer no dedicated support for these cloud platforms."

Reply:
Thanks, I like your suggestion! :)